tests/functional/test_triggers.py

```diff
@@ -0,0 +1,113 @@
+import os
+import tempfile
+import warnings
+
+from requests import exceptions
+
+from kbcstorage.base import Endpoint
+from kbcstorage.configurations import Configurations
+from kbcstorage.jobs import Jobs
+from kbcstorage.triggers import Triggers
+from kbcstorage.tables import Tables
+from kbcstorage.buckets import Buckets
+from tests.base_test_case import BaseTestCase
+
+
+class Tokens(Endpoint):
+    """
+    Testing class for obtaining token ID from token.
+    """
+
+    def __init__(self, root_url, token):
+        super().__init__(root_url, 'tokens', token)
+
+    def verify(self):
+        url = '{}/verify'.format(self.base_url)
+        return self._get(url)
+
+
+class TestEndpoint(BaseTestCase):
+    TEST_BUCKET_NAME = "kbc_trigger_test_bucket"
+    TEST_BUCKET_ID = f"in.c-{TEST_BUCKET_NAME}"
+    TEST_TABLE_NAME = "kbc_trigger_test_table"
+
+    def setUp(self):
+        self.root_url = os.getenv('KBC_TEST_API_URL')
+        self.token = os.getenv('KBC_TEST_TOKEN')
+
+        self.triggers = Triggers(self.root_url, self.token)
+        self.tables = Tables(self.root_url, self.token)
+        self.buckets = Buckets(self.root_url, self.token)
+        self.jobs = Jobs(self.root_url, self.token)
+        self.configurations = Configurations(self.root_url, self.token, 'default')
+        self.tokens = Tokens(self.root_url, self.token)
+
+        self.created_trigger_ids = []
+        # https://github.com/boto/boto3/issues/454
+        warnings.simplefilter("ignore", ResourceWarning)
+
+        self.clean()
+        self.token_id = self.tokens.verify()["id"]
+        self.buckets.create(self.TEST_BUCKET_NAME)
+        with tempfile.NamedTemporaryFile(delete=False) as tmp_file:
+            tmp_file.write(b"a,b,c\n1,2,3\n")
+        self.table_id = self.tables.create(self.TEST_BUCKET_ID, self.TEST_TABLE_NAME, tmp_file.name)
```

Comment:
for the purpose of forward compatibility you should use result id of bucket.create instead of the hardcoded in.c- something